crates/forge/src/lib.rs

```diff
@@ -129,7 +84,7 @@ pub async fn run(
 /// * `runner_params` - A struct with parameters required to run tests e.g. map with contracts
 /// * `fork_target` - A configuration of forks used in tests
 #[allow(clippy::implicit_hasher)]
-pub async fn run_with_scarb_collector(
+pub async fn run(
```

Comment:
Run internal could be inlined into this function and removed. There's no point in splitting them